virttest/vt_console.py

```diff
@@ -73,14 +76,29 @@ def __login(
         self._console.set_linesep(linesep)
         self._console.set_status_test_command(status_test_command)
 
-        is_responsive = False
+        reset_str = "Press any key to stop system reset"
         end_time = time.time() + timeout
         while time.time() < end_time:
-            if self._console.is_responsive():
-                is_responsive = True
-                break
-        if not is_responsive:
-            raise ConsoleNotResponsiveError("Console is not responsive.")
+            self._console.read_nonblocking()
+            latest_output = "\n".join(self._console.get_output().splitlines()[-50:])
```

Comment:
Here , we get output starting from [-50:], can we ensure it is the same in different arches, such as se90x , arm etc?

Reply:
You're right. Thank you for helping check this on s390. I'll get help to check on arm too.